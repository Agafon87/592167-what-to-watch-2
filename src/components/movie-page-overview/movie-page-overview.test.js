import React from "react";
import renderer from "react-test-renderer";
import MoviePageOverview from "./movie-page-overview";

it(`MoviePageOverview correctly renderer after test`, () => {
  const film = {
    starring: [],
    name: ``
  };
  const tree = renderer.create(<MoviePageOverview film={film}/>).toJSON();
  expect(tree).toMatchSnapshot();
});
