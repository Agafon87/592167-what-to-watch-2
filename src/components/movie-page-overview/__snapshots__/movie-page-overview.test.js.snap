// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePageOverview correctly renderer after test 1`] = `
Array [
  <div
    className="movie-rating"
  >
    <div
      className="movie-rating__score"
    />
    <p
      className="movie-rating__meta"
    >
      <span
        className="movie-rating__level"
      >
        Very good
      </span>
      <span
        className="movie-rating__count"
      >
         ratings
      </span>
    </p>
  </div>,
  <div
    className="movie-card__text"
  >
    <p />
    <p
      className="movie-card__director"
    >
      <strong>
        Director: 
      </strong>
    </p>
    <p
      className="movie-card__starring"
    >
      <strong>
        Starring: 
         
        and other
      </strong>
    </p>
  </div>,
]
`;
